src/back/TicketManager/src/Core/TicketManager.Core.Services/DataAccess/CoreDbContext.cs

```diff
@@ -100,6 +103,25 @@ private void ConfigureEvents(ModelBuilder modelBuilder)
             cfg.IsOptimisticConcurrent();
         });
     }
+    
+    private void ConfigureSectorReservations(ModelBuilder modelBuilder)
+    {
+        modelBuilder.Entity<SectorReservation>(cfg =>
+        {
+            cfg.HasKey(sr => sr.Id);
+            cfg.HasIndex(sr => sr.EventId).IsUnique(false);
```

Comment:
I guess we don't really need index on `EventId`, but rather on a pair `{ EventId, SectorName }`.